app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv" and len(command.split()) == 3:
+        cmd, file_original, new_file_directory = command.split()
```

Comment:
don't split command 3 times
create one separated variable for it and call it when u need it